web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1549,37 +1563,25 @@
                   getLayerInfoFromWfsCap(name, capObj, md.getUuid()),
                   olL;
               if (!capL) {
+                gnWmsQueue.removeFromQueue(url, name, map);
                 // If layer not found in the GetCapabilities
-                // Try to add the layer from the metadata
-                // information only. A tile error loading
-                // may be reported after the layer is added
-                // to the map and will give more details.
```

Comment:
If we remove that, the sample river basin of Africa will not work see https://github.com/geonetwork/core-geonetwork/issues/887#issuecomment-92259166 "the capability is not generated by GeoServer and is not containing the layer because not described in the FAO catalog" but the layer is served by this GS (which is odd). Maybe time to change the samples :)

Reply:
To me, the case makes no sense, a layer should not be called if it is not advertised. However in this PR I intended to keep this approach for WMS (for backwards compatibility). But didn't implement it for wmts/wfs.
And +1 to update the example records.

Reply:
Tested and works.

We already said we are going to update the examples. If @fxprunayre agrees, to me we can already merge this.